Makefile

```diff
@@ -10,34 +12,41 @@ dep = $(obj:.o=.d)  # one dependency file for each source
 #macos = y
 #debug = y
 #seq = y
-
-CFLAGS = -DMKL_ILP64 -m64 -I${MKLROOT}/include -I${NUMAROOT}/include -I${TBBROOT}/include -mavx2 -mfma -mf16c -fopenmp -mavx512f -Wall #-march=skylake
+SP ?=1
```

Comment:
What does SP stand for?

Reply:
SP is for "split" and it is the factor that is used to create more chunks for the parallel_for.  So you get SP * nthreads_per_socket.  We left the default at 1, although our tests did show improvements with SP=2.